cmake/sdp.cmake

```diff
@@ -17,7 +21,7 @@ function(sdp_assembly_target_sources hrt_srcs)
   foreach(hrt_src ${hrt_srcs})
     get_filename_component(hrt_dir ${hrt_src} DIRECTORY)  # directory
     get_filename_component(hrt_src_file ${hrt_src} NAME_WE)  # filename without extension
-    set(hrt_s_file "${hrt_dir}/${hrt_src_file}.s")
+    set(hrt_s_file "${hrt_dir}/${hrt_src_file}-${target_soc}.s")
```

Comment:
`target_soc` is undefined if this function is called directly.

Reply:
Added as a parameter, is that okay or further validation is necessary?